Library/Formula/jbig2dec.rb

```diff
@@ -12,9 +10,15 @@ class Jbig2dec < Formula
     sha1 "cbb4dfe055be243427210d03304c20760fb00bd7" => :lion
   end
 
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+
   def install
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    system "autoreconf", "-fvi" # error: cannot find install-sh
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}",
+                          "--disable-silent-rules", "--without-libpng"
```

Comment:
It's on my agenda for this evening. They shouldn't be using private headers for compile linkage.